### ISSUE TITLE
0crat.com seemed to have removed me from jobs but 0crat bot says otherwise

### ISSUE THREAD

**Issue by victornoel** [zerocracy/farm#2048]
Following the situation in #2030, I thought I was resigned from the jobs by 0crat and on 0crat.com I don't have any job assigned:
![image](https://user-images.githubusercontent.com/160975/58333775-7bb45a00-7e3e-11e9-8430-800d4d068abb.png)

But when I look at one of the issue I was previously assigned, I can see I'm still assigned to it according to 0crat: https://github.com/yegor256/cactoos/issues/1095#issuecomment-495430502
![image](https://user-images.githubusercontent.com/160975/58333828-9555a180-7e3e-11e9-819d-77915b851478.png)

Two things:
- I think this is a bug and this situation should be fixed, maybe it's to be fixed in the context of #2030 and/or #1885 
- What should I do with that now? If I finish the job, am I going to get some points or not? (I will try and report what happens here : )

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by g4s8** [comment]
@victornoel yes, it's a bug, 0crat should cancel all jobs on quit from project

**Comment by 0crat** [comment]
Job [#2048](https://github.com/zerocracy/farm/issues/2048) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @victornoel[/z](https://www.0crat.com/u/victornoel)
